specification/compute/Compute.Management/CapacityReservation.tsp

```diff
@@ -68,7 +68,7 @@ interface CapacityReservations {
   >;
 
   /**
-   * The operation to delete a capacity reservation. This operation is allowed only when all the associated resources are disassociated from the capacity reservation. Please refer to https://aka.ms/CapacityReservation for more details.
+   * The operation to delete a capacity reservation. This operation is allowed only when all the associated resources are disassociated from the capacity reservation. Please refer to https://aka.ms/CapacityReservation for more details. Note: Block capacity reservations cannot be deleted during their scheduled active period (between schedule start and end times).
```

Comment:
Note: Block capacity reservations cannot be deleted after it has been successfully allocated until the schedule end time.